### ISSUE TITLE
TS2315 [ERROR]: Type 'Conn' is not generic.

### ISSUE THREAD

**Issue by sharonjl** [denodrivers/postgres#281]
Running into this error with deno 1.9.1 and postgres 0.11.1

Not doing much, just importing the module gives this error.

```
error: TS2315 [ERROR]: Type 'Conn' is not generic.
  #conn!: Deno.Conn<Deno.NetAddr>;
          ~~~~~~~~~~~~~~~~~~~~~~~
    at https://deno.land/x/postgres@v0.11.1/connection/connection.ts:149:11
```

**Comment by scav** [comment]
This happens when you upgrade from 1.9.0 -> 1.9.1, as we just discovered upgrading.

**Comment by sharonjl** [comment]
Just noticed this here. https://github.com/denodrivers/postgres/issues/280

**Comment by eveningkid** [comment]
@Soremwar since #280 has been closed, could you please update this current issue here and let us know when `0.11.2` is available?

It is raising an issue with denodb and I would love to know as soon as the patch is released.

Thanks for your help, we'll be patient :)

**Comment by halvardssm** [comment]
I added a PR with a fix 💪🏻 #282
